Keypad/Keypad.js

```diff
@@ -144,7 +144,16 @@ const KeypadBase = kind({
 		disabled: PropTypes.bool,
 
 		/**
-		 * Called when a button is clicked. Includes the 'key' key in its event payload, updates the state and the input value accordingly.
+		 * Called when the value is changed.
+		 *
+		 * @type {Function}
+		 * @param {Object} event
+		 * @public
+		 */
+		onChange: PropTypes.func,
```

Comment:
If a user just import `KeypadBase` only, `onChange` will not working since it is provided by `KeypadDecorator` that is an exported HOC including `KeypadBehaviorDecorator`. What about to create a new JSDoc comment for `KeypadDecorator`?

Reply:
I am not sure I got this right. Do you mean I should add an extra line for the onChange JS Doc like:
* Note: The ready-to-use {@link agate/Keypad.KeypadBase} component needs to be wrapped in `KeypadDecorator` to support `onChange` event.

Reply:
I mean that we can add just JSDoc comments to the file so that a prop API is created for `KeypadDecorator`.
For example, you can see only JSDoc comments for documentation in the last about 100 lines in https://github.com/enactjs/agate/blob/master/Input/Input.js.

I expected something like the below. (Note that I did not check an actual docs, so please check it. 😉)
```
/**
 * Called when the input value is changed.
 *
 * @name onChange
 * @memberof agate/Keypad.KeypadDecorator
 * @type {Function}
 * @param {Object} event
 * @public
 */
```